### PR TITLE
fix and extend cluster metrics

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

This PR provides 3 metrics-related changes:

* Added cluster metrics `arangodb_load_plan_accum_runtime_msec` and `arangodb_load_current_accum_runtime_msec` to track the total time spent in `loadPlan()` and `loadCurrent()` operations.
* Fixed wrong reporting of failures in all maintenace failure counter metrics (`arangodb_maintenance_action_failure_counter`). Previously, each successful maintenance operation was reported as a failure, so the failure counters actually were counters for the number of successful maintenance actions.
* Adjusted the scale of the `arangodb_maintenance_action_queue_time_msec` to cover a more useful range.

- [x] Bug-Fix for *devel-branch* (i.e. no need for backports?)
- [x] The behavior in this PR can be (and was) *manually tested* (support / qa / customers can test it)

### Testing & Verification

This change is a trivial rework / code cleanup without any test coverage.

http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11033/

### CODE REVIEW

File: arangod/Cluster/MaintenanceFeature.cpp [hunk 1]
```diff
@@ -215,7 +215,7 @@ void MaintenanceFeature::initializeMetrics() {
                                  "Time spend execution the action [ms]"),
         metricsFeature.histogram(
             {StaticStrings::MaintenanceActionQueueTimeMs, action_label},
-            log_scale_t<uint64_t>(2, 82, 86400000, 12),
+            log_scale_t<uint64_t>(2, 82, 3600000, 12),
```
Comment (kvahed) [inline arangod/Cluster/MaintenanceFeature.cpp:218]: @maierlars could you please comment on this? i was assuming that you had made the choice deliberately? ibm clusters?

Comment (jsteemann) [reply arangod/Cluster/MaintenanceFeature.cpp:218]: All I can say is that when I looked at the metrics output, the lower bound for the smallest queue time bucket was around 42000 ms.
Which is a bit too high I think, as the queue time should be a few ms only. So in my case all actions were in that smallest bucket, and the metric was somewhat useless.

Comment (jsteemann) [reply arangod/Cluster/MaintenanceFeature.cpp:218]: Should actions be queued for more than an hour, at all?

Comment (kvahed) [reply arangod/Cluster/MaintenanceFeature.cpp:218]: it's possible. 4 core machine. and 2 big `ensureIndex` I assume

Comment (maierlars) [reply arangod/Cluster/MaintenanceFeature.cpp:218]: At that time we thought one day would be appropriate as upper bound. However, if an action is queued longer than an hour we already have a problem. So I think this is fine.

File: arangod/Cluster/MaintenanceWorker.cpp [hunk 2]
```diff
@@ -159,11 +159,11 @@ void MaintenanceWorker::nextState(bool actionMore) {
       _lastResult = _curAction->result();
       _curAction->endStats();
 
-      bool failed = _curAction->result().ok() && FAILED != _curAction->getState();
-      recordJobStats(failed);
+      bool ok = _curAction->result().ok() && FAILED != _curAction->getState();
+      recordJobStats(ok);
```
Comment (maierlars) [inline arangod/Cluster/MaintenanceWorker.cpp:163]: As far as I can tell you just renamed `failed` to `ok`. But the logic in `recordJobStats` is reverserd, so it expects `true` on failure.

Comment (jsteemann) [reply arangod/Cluster/MaintenanceWorker.cpp:163]: True :scream:

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11055/

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11063/

**Pull request merged**